forest/jasmine/traj2stats.py

```diff
@@ -140,175 +140,712 @@ def get_nearby_locations(traj: np.ndarray) -> Tuple[dict, dict, dict]:
+        d_home_1 = great_circle_dist(
+            home_lat, home_lon, temp[:, 1], temp[:, 2]
+            )
+        d_home_2 = great_circle_dist(
+            home_lat, home_lon, temp[:, 4], temp[:, 5]
+            )
+        d_home = (d_home_1 + d_home_2) / 2
+        max_dist_home = max(np.concatenate((d_home_1, d_home_2)))
+        time_at_home = sum((temp[:, 6] - temp[:, 3])[d_home <= 50])
+        mov_vec = np.round(
+            great_circle_dist(
+                temp[:, 4], temp[:, 5], temp[:, 1], temp[:, 2]
+            ),
+            0,
+        )
+        flight_d_vec = mov_vec[temp[:, 0] == 1]
+        flight_t_vec = (temp[:, 6] - temp[:, 3])[temp[:, 0] == 1]
+        pause_t_vec = (temp[:, 6] - temp[:, 3])[temp[:, 0] == 2]
+        total_pause_time = sum(pause_t_vec)
+        total_flight_time = sum(flight_t_vec)
+        dist_traveled = sum(mov_vec)
+        # Locations of importance
+        if places_of_interest is not None or save_log:
+            log_tags_temp = []
+            pause_vec = temp[temp[:, 0] == 2]
+            pause_array = np.array([])
+            for row in pause_vec:
+                if (
+                    great_circle_dist(row[1], row[2], home_lat, home_lon)
+                    > 2*place_point_radius
+                ):
+                    if len(pause_array) == 0:
+                        pause_array = np.array(
+                            [[row[1], row[2], (row[6] - row[3]) / 60]]
+                        )
+                    elif (
+                        np.min(
+                            great_circle_dist(
+                                row[1], row[2],
+                                pause_array[:, 0], pause_array[:, 1],
+                            )
+                        )
+                        > 2*place_point_radius
+                    ):
+                        pause_array = np.append(
+                            pause_array,
+                            [[row[1], row[2], (row[6] - row[3]) / 60]],
+                            axis=0,
+                        )
+                    else:
+                        pause_array[
+                            great_circle_dist(
+                                row[1], row[2],
+                                pause_array[:, 0], pause_array[:, 1],
+                            )
+                            <= 2*place_point_radius,
+                            -1,
+                        ] += (row[6] - row[3]) / 60
+
+            all_place_times = []
+            all_place_times_adjusted = []
+            if places_of_interest is not None:
+                all_place_times = [
+                    0 for _ in range(len(places_of_interest) + 1)
+                    ]
+                all_place_times_adjusted = all_place_times[:-1]
+
+            for _, row in enumerate(pause_array):
+                if places_of_interest is not None:
+                    all_place_probs = [
```

Comment:
`all_place_probs = [0] * len(places_of_interest)`
or
`all_place_probs = np.zeros_like(places_of_interest)`?

Reply:
Did this fix get reverted by accident?